xdsl/dialects/csl.py

```diff
@@ -438,6 +460,21 @@ class MemberCallOp(IRDLOperation):
 
     result = opt_result_def(Attribute)
 
+    def __init__(
+        self,
+        fname: str,
+        result_type: Attribute,
+        struct: Operation,
+        *params: SSAValue | Operation,
```

Comment:
I think I'd like this to not be a vararg, and instead be a `Sequence[SSAValue | Operation]`.

Having a vararg after a bunch of other arguments would make the code less readable at the call site imo.

Reply:
As long as it's limited to function name+type, it should hopefully be fine enough to avoid me doing cross-PR changes (:

Reply:
I'm strongly in favour of switching to `Sequence` here, as it's otherwise out of line with the other call operations...